### DISCUSSION TITLE
What's the main difference with Expo?

### DISCUSSION THREAD

**Discussion by softmarshmallow** [necolas/react-native-web#2012] [Q&A] [answered]
Can anyone help me find out the key features & difference between expo?

Is it that it can run dependent ?

**Comment by zoontek** [comment, marked as answer]
@softmarshmallow Expo for web uses react-native-web under the hood. Expo add a development environment and several modules around it.

**Reply by softmarshmallow** [reply]
Ah. that makes a lot of sense. Thanks !
